src/molecules/SplashTeam.jsx

```diff
@@ -0,0 +1,143 @@
+import React from "react";
+import SplashNav from "./SplashNav";
+import SplashTeamCard from "./SplashTeamCard";
+import styled from "styled-components";
+
+//images
+import trevorImg from "../images/trevor_clack.png";
+import timothyImg from "../images/timothy_hsu.png";
+import veraImg from "../images/vera_mendes.png";
+import markAImg from "../images/mark_artishuk.png";
+import markHImg from "../images/mark_halls.png";
+import lynnImg from "../images/lynn_baxter.png";
+import carloImg from "../images/carlo_lucido.png";
+import colinImg from "../images/colin_bazzano.png";
+import tobiasImg from "../images/tobias_reaper.png";
+import kendraImg from "../images/kendra_mckernan.png";
+import jtImg from "../images/jt_kernan.png";
+
+const Container = styled.div``;
+
+const ContentContainer = styled.div`
+  margin-top: 77px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+`;
+
+const Title = styled.h2`
+  font-family: Muli;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 24px;
+  line-height: 30px;
+  text-align: center;
+
+  color: #000000;
+`;
+
+const CardContainer = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  background: rgba(217, 217, 217, 0.3);
+  border-radius: 20px;
+  padding-top: 20px;
+  padding-bottom: 20px;
+  width: 878px;
+`;
+
+const SplashTeam = () => {
+  return (
+    <Container>
+      <SplashNav />
+      <ContentContainer>
+        <Title>The Trash Panda Team</Title>
+
+        <CardContainer>
+          <SplashTeamCard
```

Comment:
if this severely hurts our maintainability, we could probably store the images/names together and loop through them.